builtin/providers/aws/resource_aws_efs_mount_target.go

```diff
@@ -114,8 +114,8 @@ func resourceAwsEfsMountTargetCreate(d *schema.ResourceData, meta interface{}) e
 				return nil, "error", err
 			}
 
-			if len(resp.MountTargets) < 1 {
-				return nil, "error", fmt.Errorf("EFS mount target %q not found", d.Id())
+			if resp.MountTargets != nil && len(resp.MountTargets) < 1 {
```

Comment:
Could this be refactored into a helper method? We tend to repeat this process a lot